src/index.js

```diff
@@ -52,8 +52,13 @@ const release = async () => {
     await shell('git stash -u')
     await shell(`git checkout ${branch} || { git checkout -b ${branch} && git push -u origin ${branch}; }`)
     await shell(`git -c user.name='${COMMIT_NAME}' -c user.email='${COMMIT_EMAIL}' merge -`)
-    if ((await shell(`git stash list |wc -l`)) > 0) {
-      await shell('git checkout stash^3 .')
+    try {
+      if ((await shell(`git stash list |wc -l`)) > 0) {
```

Comment:
heya, i reckon get rid of the if, my guess is that stash inside the if is never being evaluated (maybe?)